pkg/function/kube_task.go

```diff
@@ -64,7 +67,8 @@ func kubeTask(ctx context.Context, cli kubernetes.Interface, namespace, image st
 		Command:      command,
 		PodOverride:  podOverride,
 	}
-
+	// Mark labels to pods with prefix `kanister.io`. Add the jobID as reference to the origin for the pod.
```

Comment:
This comment is no longer valid, either remove or correct the comment.

Reply:
Done